hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AsyncProtobufLogWriter.java

```diff
@@ -180,8 +182,10 @@ public AsyncFSOutput getOutput() {
   protected void initOutput(FileSystem fs, Path path, boolean overwritable, int bufferSize,
     short replication, long blockSize, StreamSlowMonitor monitor)
     throws IOException, StreamLacksCapabilityException {
+    boolean noLocalWrite =
```

Comment:
Better add this as a parameter of the initOutput method? So we do not need to write the similar code twice in AsyncProtobufLogWriter and ProtobufLogWriter.

Reply:
done!